packages/create-llama/questions.ts

```diff
@@ -233,30 +233,31 @@ export const askQuestions = async (
       program.engine = engine;
       preferences.engine = engine;
     }
-  }
 
-  if (program.engine !== "simple" && !program.vectorDb) {
-    if (ciInfo.isCI) {
-      program.vectorDb = getPrefOrDefault("vectorDb");
-    } else {
-      const { vectorDb } = await prompts(
-        {
-          type: "select",
-          name: "vectorDb",
-          message: "Would you like to use a vector database?",
-          choices: [
-            {
-              title: "No, just store the data in the file system",
-              value: "none",
-            },
-            { title: "MongoDB", value: "mongo" },
-          ],
-          initial: 0,
-        },
-        handlers,
-      );
-      program.vectorDb = vectorDb;
-      preferences.vectorDb = vectorDb;
+    if (program.engine !== "simple" && !program.vectorDb) {
+      if (ciInfo.isCI) {
+        program.vectorDb = getPrefOrDefault("vectorDb");
+      } else {
+        const { vectorDb } = await prompts(
+          {
+            type: "select",
+            name: "vectorDb",
+            message: "Would you like to use a vector database?",
+            choices: [
+              {
+                title: "No, just store the data in the file system",
+                value: "none",
+              },
+              { title: "MongoDB", value: "mongo" },
+              { title: "PostgreSQL", value: "pg" },
```

Comment:
Ensure that `pg` can't be selected for Python templates, either:
1. get it dynamically from the folder or (might not work with `ncc build`)
2. do a lookup of choices that is language-specific